packages/dds/merge-tree/package.json

```diff
@@ -30,6 +30,8 @@
     "format": "npm run prettier:fix",
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
+    "perf": "cross-env FLUID_TEST_VERBOSE=1 mocha \"dist/**/*.tests.js\" --node-option unhandled-rejections=strict,expose-gc --exit -r node_modules/@fluidframework/mocha-test-setup --perfMode --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js --timeout 30000",
```

Comment:
Ah, I'd also point back to this, if we want to have the new tests run as part of the Performance Benchmarks pipeline. If they're still in flux and just for running locally, this can come later, but if we want them to run on every commit, we'll want to finish the extra steps from the linked docs.